docs/release-notes.rst

```diff
@@ -36,6 +36,15 @@ Changes and New Features
 
    * - Version
      - Description
+   * - 25.4.0
+     - | - Added support for NVIDIA NIC Configuration Operator deplyment through NicClusterPolicy CR as well as Helm chart.
+       | - Integrate NVIDIA Network Operator with NVIDIA Maintenance Operator for DOCA driver upgrade.
+       | - Added support for NVIDIA Spectrum-X Operator deployment:
```

Comment:
I propose not to go into details on Spectrum-X Operator.
Just this for example, and move it to the end.
```
Added support for NVIDIA Spectrum-X Operator deployment (Tech Preview)
```